docs/CMakeLists.txt

```diff
@@ -9,12 +9,14 @@
 # Setup dependencies
 ########################################################################
 find_package(Doxygen)
+find_package(MathJax2)
 
 ########################################################################
 # Register component
 ########################################################################
 include(GrComponent)
-GR_REGISTER_COMPONENT("doxygen" ENABLE_DOXYGEN DOXYGEN_FOUND)
+GR_REGISTER_COMPONENT("doxygen" ENABLE_DOXYGEN
```

Comment:
We probably shouldn't be introducing this line break?

Reply:
Oops, possibly a leftover of a previous discarded change.